python/ray/serve/deployment_graph.py

```diff
@@ -35,7 +32,9 @@ def __reduce__(self):
 
     def remote(self, *args, **kwargs):
         if self.dag_node is None:
+            from ray.serve.pipeline.json_serde import dagnode_from_json
```

Comment:
i moved it to init in earlier PR because i think putting minimal work on the `remote()` path is probably better, might not be a big deal here given will only run it once.